src/sentry/api/serializers/models/event.py

```diff
@@ -357,7 +352,7 @@ def _get_perf_problem(self, attrs):
         converted_problem = convert_dict_key_case(perf_problem, snake_to_camel_case)
         issue_type = perf_problem.get("type")
         if issue_type in [type.value for type in GroupType]:
-            converted_problem["issueType"] = GROUP_TYPE_TO_TEXT.get(GroupType(issue_type), "Issue")
+            converted_problem["issueType"] = GroupType(issue_type).name.lower()
```

Comment:
Slight chance for backwards compatibility issue due to a changed response. NBD Here.

Reply:
Its a nonissue this property of the response was only introduced yesterday